docs/en/observability/observability-ui.asciidoc

```diff
@@ -1,33 +1,97 @@
+The *Overview* page contains charts displaying analytics relating to the components
+that help you make your systems observable; logs, metrics, APM, and uptime data.
 
-The dashboard is comprised of four components to help you make your environments 
-observable; APM, logs, metrics, and uptime. 
+Additionally, the alerts chart keeps you informed of any issues that you may need
+to resolve quickly.
 
-=== Services, transactions, and error rates
+[float]
+[[view-services-and-transactions]]
+== Services and transactions
 
-coming[a future update]
+The *APM* chart displays the total number of services running within your environment
+and the total number of transactions per minute that were captured by the Elastic APM
+agent instrumenting those services.
 
-=== Log rates
+To drill-down into this APM data, click {kibana-ref}/services.html[*View in app*].
 
-coming[a future update]
+//TODO: what are the specific metric fields?
 
-=== System metrics
+[role="screenshot"]
+image::images/apm.png[APM summary]
 
-coming[a future update]
+[float]
+[[view-log-rates]]
+== Log rates
 
-=== Systems availability 
+The *Logs* chart helps you to detect and inspect possible log anomalies across each of
+your ingested log sources. The visualization helps you determine if the log rate is outside
+of your expected bounds, and therefore could be considered anomalous. Any drop in the log
+rate could suggest a system has stopped responding, or a spike could denote a DDos attack.
 
-coming[a future update]
+To drill-down into the logs, click {logs-guide}/logs-app-overview.html[*View in app*].
 
-=== Alerts 
+//TODO: what are the specific metric fields?
 
-coming[a future update]
+[role="screenshot"]
+image::images/log-rate.png[Log rates]
+
+[float]
+[[view-system-metrics]]
+== System metrics
+
+The *Metrics* chart provides you with a quick glance of the performance metrics
+aggregated on all of your running hosts. The metrics help you to evaluate the efficiency
+of hosts and determine whether there is any impact of resource consumption for your end-users.
+
+CPU usage, memory usage, and disk used metrics enable you to identify possible memory leaks,
+and whether disk resources need to be added or redistributed. Inbound and outbound traffic
+metrics provide you with a visualization of any possible spikes to help determine whether
+any hosts are choking your network. 
+
+To drill-down into the performance metrics, click {metrics-guide}/metrics-app-overview.html[*View in app*].
+
+//TODO: what are the specific metric fields?
+
+[role="screenshot"]
+image::images/metrics-summary.png[Metrics summary]
+
+[float]
+[[view-systems-availability]]
+== Systems availability 
+
+To help you react quickly to availability issues before they affect your users, the *Uptime*
+provides you with a snapshot of the overall availability of your hosts, network devices, or third-party
+services. You can view the total number of detected monitors, and the state of each one; whether
```

Comment:
This sentence is confusing to me: "You can view the total number of detected monitors, and the state of each one; whether they are up or down within a specific timeframe."

Reply:
@bmorelli25 Maybe this reads better. Based on the last check reported by Heartbeat, you can view the total number of detected monitors, along with the number of monitors in an up or down state.